From 0797773d6d17c6def1d9373175c0d4b35bac5e17 Mon Sep 17 00:00:00 2001
From: Ihar Hrachyshka <ihar.hrachyshka@gmail.com>
Date: Sat, 7 Sep 2024 14:08:23 -0400
Subject: [PATCH] activate-system: Activate user environment too

---
 modules/services/activate-system/default.nix | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/modules/services/activate-system/default.nix b/modules/services/activate-system/default.nix
index c41d963..a186086 100644
--- a/modules/services/activate-system/default.nix
+++ b/modules/services/activate-system/default.nix
@@ -42,5 +42,10 @@ in
       serviceConfig.KeepAlive.SuccessfulExit = false;
     };
 
+    launchd.user.agents.activate-user = {
+      command = "/run/current-system/activate-user";
+      serviceConfig.RunAtLoad = true;
+      serviceConfig.KeepAlive.SuccessfulExit = false;
+    };
   };
 }
-- 
2.46.0

